docs/src/main/asciidoc/kafka.adoc

```diff
@@ -2052,7 +2053,75 @@ class BaristaTest {
 <3> Retrieve the outgoing channel (`beverages`) - the channel must have been switched to in-memory in the test resource.
 <4> Use the `send` method to send a message to the `orders` channel.
 The application will process this message and send a message to `beverages` channel.
-<5> Use the `received` method on `beverages` channel to check the messages produced by the application.
+<5> Use the `received` method on `beverages` channel to check the messages produced by the application. The `await` is imported from https://github.com/awaitility/awaitility[awaitility].
```

Comment:
I think we should add the imports to the examples instead. That's why we try to do elsewhere as it can be hard to get it right if you don't have the imports.

Note that this is orthogonal to your PR but maybe you could take care of it in a separate commit, instead of adding this sentence?

Reply:
done